src/fetchPost.js

```diff
@@ -0,0 +1,365 @@
+            .join(" ")}
+          </section>
+          <section class="single-post__description">
+            <section class="single-post__description-title">توضیحات</section>
+
+            <section class="single-post__description-text">${description.replace(
+              "\\n",
+              "<br/>"
+            )}</section>
+          </section>
+          <section class="single-post__tag-box">
+          ${chips
+            .map(
+              (chip) => `<button class="single-post__tag">${chip.text}</button>`
+            )
+            .join(" ")}
+            
+          </section>
+        </section>
+        <section class="single-post__left-box">
+          <section class="single-post__image-container square-image-container ${
+            !hasImage ? "hidden" : ""
+          }">
+          <section class="square-image-container__wrapper">
+            <img
+              class="square-image-container__image single-post__image"
+              src="${hasImage ? images[0].image.url : ""}"
+            />
+          </section>
+          </section>
+          <section class="single-post__thumbnail-container ${
+            !hasImage ? "hidden" : ""
+          }">
+            ${images
+              .map(
+                (image) => `
+            <section class="single-post__thumbnail square-image-container">
+            <div class="square-image-container__wrapper">
+              <img
+                class="square-image-container__image"
+                src="${image.image.url}"
+              />
+            </div>
+            </section>`
+              )
+              .join(" ")}
+          </section>
+          <section class="single-post__note-container">
+            <textarea id="note" class="single-post__note" type="textbox" placeholder="یادداشت">${
+              notes ? (notes[postID] ? notes[postID] : "") : ""
+            }</textarea>
+            <section class="single-post__note-warning">یادداشت تنها برای شما قابل دیدن است و پس از حذف آگهی، پاک خواهد شد.
+            </section>
+          </section>
+          <a class="single-post__safe-buy iconed-link" href="https://support.divar.ir/b/support-users/fa/kb/articles/article-46">
+                <img class="iconed-link__icon" src="${
+                  require("./icons/svgs/check-circle-o.svg").default
+                }" />
+                <span class="iconed-link__text">راهنمای خرید امن</span>
+          </a>
+          <hr class="single-post__row-separator"/>
+          <a class="single-post__report iconed-link" href="https://support.divar.ir/b/support-users/fa/kb/articles/article-46">
+                <img class="iconed-link__icon" src="${
+                  require("./icons/svgs/info-circle-o.svg").default
+                }" />
+                <span class="iconed-link__text">ثبت تخلف و مشکل آگهی</span>
+          </a>
+          <hr class="single-post__row-separator"/>
+        </section>
+      </article>
+      </main>`.concat(container.innerHTML);
```

Comment:
You could've created a container for the HTML that is going to be inserted. Moving the main tag to HTML instead of inserting it from here would've saved you from needing to do this concat here.